server/src/main/java/org/jboss/as/server/parsing/StandaloneXml.java

```diff
@@ -46,14 +48,26 @@
  */
 public class StandaloneXml extends CommonXml {
 
-    private final ExtensionXml extensionXml;
-    private final ExtensionRegistry extensionRegistry;
+    private final StandaloneXml_4.ParsingOptions parsingOptions;
```

Comment:
These ParsingOptions shouldn't be in StandaloneXml_4. That class is not available for outside use.